app/views/login.html

```diff
@@ -1 +1,17 @@
-<h1>Hi this is login page</h1>
+<form>
+  <div class="form-alert">
+    username not found
+  </div>
+  <div class="form-group">
+    <label for="username">Username</label>
+    <input id="username" type="text" required placeholder="ashketchum10">
```

Comment:
actually it's kinda confusing to have a maxlength on creation b/c you might not be looking see that it didn't fit. nvm.